accepted/2021/experimental-package-shading/experimental-package-shading.md

```diff
@@ -0,0 +1,339 @@
+# Experiment: NuGet Package Shading
+
+**Owner** [Mikayla Hutchinson](https://github.com/mhutch)
+
+## Introduction
+
+Producer-side package shading is an experimental feature that allows a NuGet package author to "shade" a dependency: embed a renamed copy of it in their package. This ensures that consumers of the package get the exact same version that the package author intended, regardless of any other direct or indirect references to that dependency.
+
+**The major downside to producer-side shading** is that the shaded assemblies are included in the app even when they are identical or completely compatible with other shaded or non-shaded copies of the same dependency. This forced redundancy is a problem for scenarios where app size and memory use is highly important such as mobile apps and client-side web apps, **so producer side shading is not intended to ever become non-experimental or recommended for general use**.
```

Comment:
I would put a note in here that in some cases, it's impossible for a package author to know ahead of time that their package could cause conflicts.  Sometimes it's the combination of random packages that cause conflicts, so it doesn't really make sense for the producer to have to handle it.